core/controllers/ingest.py

```diff
@@ -313,3 +317,15 @@ def save_submission_file(excel_file, submission_id):
     submission.submission_file = excel_file.stream.read()
     db.session.add(submission)
     db.session.commit()
+
+
+def remove_defunct_organisations():
```

Comment:
can we rename this function? Defunct is a bit of a loaded term, like the data has changed. We could maybe change it to something more explicitly descriptive. I think the correct database term is unreferenced row or dead row. Probably unreferenced is better